### ISSUE TITLE
funds disappeared after recovery firefly with recovery phrase

### ISSUE THREAD

**Issue by Ab1960** [iotaledger/firefly#1864]
### Issue description

Used recovery phrase voor firefly on windows 10 on laptop. The funds did not show up. Used other nodes, used balance finder and still nothing. Tried version 1.2 and 1.3. On discord there was no solution

### Firefly version

version 1.2 and 1.3

### OS details

Windows 10

### Can the issue be reliably reproduced?

Yes

### Steps to reproduce the issue

1.
2.
3.
...
Steps to reproduce the issue
Install Firefly 1.2.0
Open Firefly 1.2.0
Create a profile
Migrate or restore a wallet
I have a  backup and recovery phrase
Setting up pincode
Done

### Expected behaviour

Expected to show balance after migration. 

### Actual behaviour

There is no balance and there is no history in addresses.

**Comment by MiKeF05** [comment]
Which node are you connected to?

**Comment by Ab1960** [comment]
https://chrysalis-nodes.iota.org and cafe
I also tried one, that was recomended on discord
, but nothing

**Comment by MiKeF05** [comment]
Anything in the error log?

**Comment by Ab1960** [comment]
I am out of the house, i Will answer when i am back. But i didnot see any
error

Op do 23 dec. 2021 12:06 schreef MiKeF05 ***@***.***>:

**Comment by MiKeF05** [comment]
Maybe you can join the iota discord server and go to the help channel. I think this is a better way to solve your issue ;-)

**Comment by Ab1960** [comment]
I was there, there was no solution.

Op do 23 dec. 2021 12:28 schreef MiKeF05 ***@***.***>:

**Comment by MiKeF05** [comment]
Are you sure you are using the correct 24 words for the recovery? And you have already migrated to chrysalis and the balance appeared?

**Comment by Mke91** [comment]
How often you have clicked "balance finder" ? Could happen that you have to click it multiple times to enhance the range of searchinng

**Comment by Ab1960** [comment]
Yes I migrated in the beginning this year and made a back up. Now i want
tot acces de wallet for the staking. So i reinstalled the wallet but there
are no tekens now

Op do 23 dec. 2021 12:40 schreef MiKeF05 ***@***.***>:

**Comment by Ab1960** [comment]
About 50 times

Op do 23 dec. 2021 12:42 schreef Mke91 ***@***.***>:

**Comment by MiKeF05** [comment]
So you now have an empty profile with no balance and no history?

**Comment by Ab1960** [comment]
Yes no balance and no history. I saw a similar problem with someone on
gridhub with a ledger. He couldnot see it either after recovering firefly
with a ledger.

Op do 23 dec. 2021 12:46 schreef MiKeF05 ***@***.***>:

**Comment by MiKeF05** [comment]
Do you also have tried it with the stronghold backup?

**Comment by Ab1960** [comment]
Yes. The same problemen.

Op do 23 dec. 2021 13:01 schreef MiKeF05 ***@***.***>:

**Comment by azad234** [comment]
> ### Fehlerbeschreibung
> Verwendete Wiederherstellungsphrase für Glühwürmchen unter Windows 10 auf dem Laptop. Das Geld ist nicht aufgetaucht. Andere Knoten verwendet, Balance-Finder verwendet und immer noch nichts. Versuchte Version 1.2 und 1.3. Bei Zwietracht gab es keine Lösung
> 
> ### Firefly-Version
> Version 1.2 und 1.3
> 
> ### Betriebssystemdetails
> Windows 10
> 
> ### Kann das Problem zuverlässig reproduziert werden?
> ja
> 
> ### Schritte zum Reproduzieren des Problems
> 
> ... Schritte zum Reproduzieren des Problems Firefly 1.2.0 installieren Firefly 1.2.0 öffnen Profil erstellen Wallet migrieren oder wiederherstellen Ich habe eine Sicherungs- und Wiederherstellungsphrase PIN-Code einrichten Fertig
> 
> ### Erwartetes Verhalten
> Voraussichtlicher Ausgleich nach der Migration.
> 
> ### Tatsächliches Verhalten
> Es gibt keinen Saldo und keine Adressenhistorie.
> 
> ### Fehler
> _Keine Antwort_

I have the same problem!
After migrating to 24 words
Is my balance 0

**Comment by MiKeF05** [comment]
@azad234 Which steps have you done before? Did you have a profile before or why did you want to restore your profile?

**Comment by Ab1960** [comment]
I had created my profile and wallet after the migration in April 2021 on my
desktop. I did a few transactions to another wallet to test the operation.
And I made a backup of my wallet. My desktop is in the Netherlands and in
November 2021 I went to Spain. Because there was talk of the stake soon, I
took a backup and the 24 word phrase with me to Spain. When the date of the
stake became known I tried to restore the wallet on my laptop. After the
restore the balance was 0. I am going back to the Netherlands in mid
February and then I hope the wallet on my desktop is still working. Only
then I miss the stake thing.

I have found a transaction in another wallet with the wallet I wanted to
restore. On the website explorer.iota.org I can see that on the day of the
transaction (April 2021) the balance in my wallet was still correct. Only I
do not know how I can check from this transaction whether my funds are
still in the wallet. I have asked on discord, but no response

Op zo 2 jan. 2022 23:56 schreef MiKeF05 ***@***.***>:
